cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -56,16 +60,19 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
             |If a user has not been granted `ACCESS` privilege to any databases, the command can still be executed but will only return the `system` database, which is always visible.
             |""".stripMargin)
           }
+      p(
+        """In this example, the detailed information for a particular database can be seen using the command `SHOW DATABASE name YIELD *`. When a `YIELD`
+          |is specified, the full set of columns is returned.
+          |""".stripMargin)
+      query("SHOW DATABASE movies YIELD *", assertDatabaseShown("movies")) {
+        resultTable()
+      }
```

Comment:
I wonder if the example on (now) line 84 also should be moved up since both this and the next examples uses YIELD and it sounds like that one introduces it

Reply:
I think this is okay actually, it introduced it in the context of sorting and filtering which is different to what we're doing here.